validator/accounts/wallet_create.go

```diff
@@ -35,10 +37,6 @@ func CreateAndSaveWalletCli(cliCtx *cli.Context) (*wallet.Wallet, error) {
 	if err != nil {
 		return nil, err
 	}
-	if keymanagerKind == keymanager.Imported {
-		log.Warning("Creating an imported wallet does not provide anything beneficial. " +
-			"This option will be removed in v1.0.0. Please use `validator accounts import` instead.")
-	}
```

Comment:
How is this relevant to your dynamic key reloading changes? 
Should it be removed? cc: @rauljordan

Reply:
Why are there changes in this file at all? I thought this PR was to fix operations with dynamic key reloading on existing wallets.

Reply:
This was removed because we now allow empty wallets. Previously it did not make sense to have one. It has been some time since this change, so I don't remember the exact details. If I'm not mistaken, it was a request from Bison Trails.

@rauljordan Can you confirm?

Reply:
Staking as a service providers prefer to create an empty wallet and then dynamically import keys as needed in their cloud deployments. We should probably move that to a separate PR IMO to keep the diff small, however.

Reply:
Agreed that we can move it to a separate PR and merge it before this one.